content/templates/implementations.md

```diff
@@ -50,20 +51,20 @@ image: null
 
 ##### Web
 
-- <a href="https://github.com/airbnb/lottie-web" target="_blank">lottie-web</a> - official
-- <a href="https://github.com/LottieFiles/lottie-player" target="_blank">lottie-player</a> - by @lottiefiles
-- <a href="https://github.com/LottieFiles/lottie-react" target="_blank">react-lottie-player</a> - by @lottiefiles
-- <a href="https://github.com/gamote/lottie-react" target="_blank">lottie-react</a> - by @gamote
-- <a href="https://github.com/LottieFiles/svelte-lottie-player" target="_blank">svelte-lottie-player</a> - by @lottiefiles
-- <a href="https://github.com/chenqingspring/vue-lottie" target="_blank">vue-lottie</a> - by @chenqingspring
-- <a href="https://github.com/chenqingspring/ng-lottie" target="_blank">ng-lottie</a> - Lottie for Angular by @chenqingspring
+- <a href="https://github.com/airbnb/lottie-web" target="_blank">lottie-web</a> by Airbnb
```

Comment:
I'm not super familiar with the origins of the project, so please correct me if I'm wrong: my understanding is that unlike the other players under Airbnb (which were originally developed by Airbnb employees), LottieWeb was developed independently by Hernan is just hosted under the org.  If that's the case, maybe skip this particular attribution?

Reply:
thanks for the correction, fixed